simulation.py

```diff
@@ -45,8 +45,8 @@
     
     
     #create some send events    
-    for i in range(3):
-        client.udt_send(2, 'Sending a much larger message this time for the first part of the thing %d' % i)
+    for i in range(1):
```

Comment:
This was changed to 1 for testing implementation. It still works when sending multiple messages